view/templates/settings/display.tpl

```diff
@@ -33,28 +33,30 @@
 	{{include file="field_select.tpl" field=$platform_icon_style}}
 	{{include file="field_checkbox.tpl" field=$embed_remote_media}}
 	{{include file="field_checkbox.tpl" field=$embed_media}}
-	<h2>{{$timeline_title}}</h2>
-	{{$timeline_explanation}}
-	<table class="table table-condensed table-striped table-bordered">
-	<thead>
-	<tr>
-		<th>{{$timeline_label}}</th>
-		<th>{{$timeline_descriptiom}}</th>
-		<th>{{$timeline_enable}}</th>
-		<th>{{$timeline_bookmark}}</th>
-	</tr>
-	</thead>
-	<tbody>
-	{{foreach $timelines as $t}}
-		<tr>
-			<td>{{$t.label}}</td>
-			<td>{{$t.description}}</td>
-			<td>{{include file="field_checkbox.tpl" field=$t.enable}}</td>
-			<td>{{include file="field_checkbox.tpl" field=$t.bookmark}}</td>
-		</tr>
-	{{/foreach}}
-	</tbody>
-	</table>
+	<h2 tabindex="0">{{$timeline_title}}</h2>
+	<p tabindex="0">{{$timeline_explanation}} {{$sortable}}</p>
+	<h3 tabindex="0">{{$timeline_enable}}</h3>
+	<div class="select timelines-widget sortable">
+		<input type="hidden" id="widget_timelineorder" name="widget_timelineorder" value=""/>
+		{{foreach $timelines as $t}}
+			{{include file="field_checkbox.tpl" field=$t.enable}}
+		{{/foreach}}
+		<div class="settings-submit-wrapper">
+			<input type="hidden" name="widget_timeline_reset" value="0"/>
+			<input type="checkbox" id="widget_timeline_reset" name="widget_timeline_reset" value="1"/> <label for="widget_timeline_reset">{{$reset_label}}</label>
```

Comment:
I would like to stick with our existing checkbox template instead of hardcoding the HTML elements here. Is there something that the template can't do?